namespaces/_default_.yml

```diff
@@ -83,51 +83,48 @@ _default_:
         CRITICAL -> crit, ERROR -> err, ...., DEBUG -> debug
 
     - name: message
-      type: string
-      index: analyzed
-      doc_values: false
+      type: text
+      index: true
```

Comment:
index: true means not_analyzed?  How do we tell Elasticsearch that we do want message to be analyzed?

Reply:
Or is analyzed the default for type: text?

Reply:
See https://github.com/ViaQ/elasticsearch-templates/issues/68

> The index option controls whether field values are indexed. It accepts true or false. Fields that are not indexed are not queryable.